ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -224,7 +224,7 @@ export default class ConfirmPageContainerContent extends Component {
                 className="actionable-message--warning"
                 message={
                   <Typography variant={TYPOGRAPHY.H7} align="left">
-                    {t('insufficientCurrency', [nativeCurrency, networkName])}
+                    {t('insufficientCurrency', [nativeCurrency, networkName])}{' '}
```

Comment:
Could this be merged into one message? Splitting adjacent text into multiple messages is generally bad for localization because in some locales the messages might be changed by the surrounding context. Even the number of sentences might differ between locales.

Reply:
Yea I don't see why not... Not sure why it was done this way originally.